DiceClass

MojoOno opened the pull request:
Added diceClass

MojoOno committed:
Added Dice class with fitting dDices

AndyTheDragon reviewed (changes requested):
Hvorfor er der en klasse inde i en klasse?
Kan man nøjes med en Random random = new Random()

MojoOno committed:
Rettede Dice klassen, fra at være lavet så hurtigt som muligt, til at være gennemtænkt en anelse mere.
Dvs. fjernede static fra roll metoderne, fjernede duplicate klasse og instantierede vores Random i klassen, istedet for i metoderne.
